.github/workflows/build_deploy_exm_fe.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Deploy EXM FE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build Angular application
+      run: npm run build --prod
+
+    - name: Terraform Output - Get VM IP
+      id: get-vm-ip
+      run: echo "::set-output name=vm_ip::$(terraform output -raw public_ip)"
+
+    - name: SSH into VM and install Nginx
+      run: |
+        ssh -o StrictHostKeyChecking=no $TF_VAR_admin_username@${{ steps.get-vm-ip.outputs.vm_ip }} << 'EOF'
```

Comment:
TF_VAR_admin_username probabil va fi numit doar admin_username, prefixul de TF_VAR se pune pentru variabilele care sunt folosite in terraform, nu este nevoie aici